docs/data/intake-api/generated/transaction/null_values.json

```diff
@@ -31,14 +31,16 @@
             "timestamp": "2017-05-30T18:53:27.154Z",
             "result": "200",
             "context": null,
-            "spans": null
+            "spans": null,
+            "sampled": false
```

Comment:
please rset to `null` to test that you actually allow null values (this test will fail, as there is a null option missing in the spec).